apps/aehttp/test/aehttp_hyperchains_SUITE.erl

```diff
@@ -375,11 +377,29 @@ spend_txs(Config) ->
     seed_account(pubkey(?LISA), 100000003 * ?DEFAULT_GAS_PRICE, NetworkId),
 
     produce_cc_blocks(Config, 1),
-    %% Give some time to sync nodes and remove Txs from pool
     {ok, []} = rpc:call(?NODE1_NAME, aec_tx_pool, peek, [infinity]),
     %% TODO check that the actors got their share
     ok.
 
+check_blocktime(_Config) ->
+    {ok, TopBlock} = rpc(?NODE1, aec_chain, top_key_block, []),
+    check_blocktime_(TopBlock).
+
+check_blocktime_(Block) ->
+    case aec_blocks:height(Block) >= 1 of
+        true ->
+            {ok, PrevBlock} = rpc(?NODE1, aec_chain, get_block, [aec_blocks:prev_key_hash(Block)]),
+            Time1 = aec_blocks:time_in_msecs(Block),
+            Time2 = aec_blocks:time_in_msecs(PrevBlock),
+            [ ct:pal("Blocktime not respected KB(~p) at ~p and KB(~p) at ~p",
+                     [aec_blocks:height(Block), Time1, aec_blocks:height(PrevBlock), Time2])
+              || Time1 - Time2 < ?CHILD_BLOCK_TIME ],
+            ?assertMatch(Diff when Diff >= ?CHILD_BLOCK_TIME, Time1 - Time2),
```

Comment:
Once we actually add _slots_, yes the checks should be extended! 

Right now there are no such guarantees, especially since the test framework explicitly halts mining.